src/main/java/com/wanderersoftherift/wotr/modifier/Modifier.java

```diff
@@ -20,22 +19,29 @@
 
 public class Modifier {
     public static final Codec<Modifier> DIRECT_CODEC = RecordCodecBuilder.create(inst -> inst.group(
-            ModifierTier.CODEC.listOf().fieldOf("tiers").forGetter(Modifier::getModifierTierList)
+            ModifierTier.CODEC.listOf().fieldOf("tiers").forGetter(Modifier::getModifierTierList),
+            Codec.INT.optionalFieldOf("color", 0xFFFFFF).forGetter(Modifier::getColor)
     ).apply(inst, Modifier::new));
     public static final Codec<Holder<Modifier>> CODEC = RegistryFixedCodec.create(WotrRegistries.Keys.MODIFIERS);
     public static final StreamCodec<RegistryFriendlyByteBuf, Holder<Modifier>> STREAM_CODEC = ByteBufCodecs
             .holderRegistry(WotrRegistries.Keys.MODIFIERS);
 
     private final Map<Integer, ModifierTier> modifierTiers;
+    private final int color;
```

Comment:
There is possibly an argument for using Style here, so that each Modifier can define its own style rather than just color. Depends whether we're likely to want to use any other parts of the Style - are there any modifiers that would be bold, or italic, or obfuscated, or use a different font, or are these things we'ld want to be able to programmatically control for specific situations.